types/lib/operators.d.ts

```diff
@@ -18,6 +18,7 @@ declare const Op: {
   readonly in: unique symbol;
   readonly iRegexp: unique symbol;
   readonly is: unique symbol;
+  readonly join: unique symbol;
```

Comment:
Op.join only appears to be referenced a single time, I'm not sure if this is internal only.